apps/meteor/app/livechat/server/livechat.ts

```diff
@@ -1,11 +1,37 @@
 import url from 'url';
 
+import jsdom from 'jsdom';
 import { WebApp } from 'meteor/webapp';
 
 import { settings } from '../../settings/server';
 import { addServerUrlToIndex } from '../lib/Assets';
 
-const indexHtmlWithServerURL = addServerUrlToIndex((await Assets.getTextAsync('livechat/index.html')) || '');
+let indexHtmlWithServerURL = addServerUrlToIndex((await Assets.getTextAsync('livechat/index.html')) || '');
+const domParser = new jsdom.JSDOM(indexHtmlWithServerURL);
+const doc = domParser.window.document;
+const head = doc.querySelector('head');
+const body = doc.querySelector('body');
+
+// TODO IMPROVE CONDITIONS CHECK
+const liveChatAdditionalScripts = settings.get<string>('Livechat_AdditionalWidgetScripts');
+if (liveChatAdditionalScripts) {
+	liveChatAdditionalScripts.split(',').forEach((script) => {
+		const scriptElement = doc.createElement('script');
+		scriptElement.src = script;
+		head?.appendChild(scriptElement);
+		body?.appendChild(scriptElement);
```

Comment:
Do we need to put the same script in both?

Reply:
There may be edge cases where there is no head on the DOM where we should load the script in the body element, but i agree that we should not load the script twice. I will proceed with only appending it to in the head of the document.